src/Docker/Runner.php

```diff
@@ -507,6 +552,9 @@ private function runImages(
         foreach ($images as $priority => $image) {
             if ($image->isMain()) {
                 $stateFile->createStateFile();
+                if (!is_null($configId) && in_array('artifacts', $tokenInfo['owner']['features'])) {
+                    $this->downloadArtifacts($artifacts, $image->getConfigData()['artifacts']);
```

Comment:
Tady chybi isset. A je to tak hluboko pres N private metod ze to zadny test nechytil.
Ty artifacts jsou optional a ne povinne.

Reply:
JJ, uz som to fixol